otel/otel.go

```diff
@@ -25,6 +25,8 @@ var (
 
 	stopper   func()
 	stopperMx sync.Mutex
+
+	stopReconfiguration bool
```

Comment:
Maybe an atomic bool here just in case another goroutine happens to call Configure? Otherwise LGTM